quarto/table-templates/template-table_03.qmd

```diff
@@ -1,62 +1,164 @@
 ---
```

Comment:
I think this file can be archived altogether since we're not supporting rtables anymore and there are no functions/parameters